enhancements/update/versioning-cincinnati-api.md

```diff
@@ -0,0 +1,116 @@
+---
+title: versioning-cincinnati-api
+authors:
+- "@pratikmahajan"
+  reviewers:
+- @openshift/openshift-team-cincinnati-maintainers
+- @openshift/openshift-team-cincinnati
+- @openshift/team-cincinnati-operator
+  approvers:
+- @crawford
+- @LalatenduMohanty
+- @sdodson
+  creation-date: 2021-08-20
+  last-updated: 2021-08-20
+  status: implementable
+  see-also:
+-
+replaces: []
+superseded-by: []
+---
+
+# Versioning cincinnati api and json schema
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+
+## Summary
+As of today, we declare the schema version in Cincinnati Graph URI. We currently, do not embed the version information
+in json schema. The enhancement will embed the version information in the json schema. This has the benefit of being 
+easy to implement and to pass around if we ship Cincinnati JSON over non-HTTP channels. Enhancement will also implement
+media type versioning while requesting the graph e.g. `ContentType: application/vnd.redhat.cincinnati.v1+json`. 
+The move to supplying requested version in media type field will require us to move towards a generic un-versioned Graph
+URI, with the move, we'll still support old URI for foreseeable future. 
+
+## Motivation
+
+### Goals
+1. Embed version information in json schema.
+2. Implement media type versioning for clients requesting a specific schema version of cincinnati
```

Comment:
nit: when discussing the protocol (in a non-slug form), I think the canonical casing is "Cincinnati" (e.g. [here][1]).

[1]: https://github.com/openshift/cincinnati/blame/d6a64f81a50e68e5feae52df9d5ae23fa7ab6dad/docs/design/cincinnati.md#L19